coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalHierarchyServiceImpl.java

```diff
@@ -342,10 +344,44 @@ public void removeFromHierarchy(DevelopmentProposal childProposal) throws Propos
             }
             hierarchyProposal.setHierarchyOriginatingChildProposalNumber(lowestProposalNumber);
             dataObjectService.save(childProposal);
+            dataObjectService.save(hierarchyProposal);
```

Comment:
Sorry, just seeing this. So I actually thought some of the issues I was seeing was because the hierarchyProposal was not being saved in this sequence of code. I am working on more hierarchy stuff. I can try to take it out and test again.